golang/cosmos/app/export.go

```diff
@@ -46,7 +46,8 @@ func (app *GaiaApp) ExportAppStateAndValidators(
 
 // prepare for fresh start at zero height
 // NOTE zero height genesis is a temporary feature which will be deprecated
-//      in favour of export at a block height
+//
+//	in favour of export at a block height
```

Comment:
Clean up the formatting, and consider godoc-standard function comment formatting.